R/item_deletion_h.R

```diff
@@ -176,58 +171,37 @@ item_deletion_h <- function(propsel = NULL,
     return_items <- user_specified_items
   }
   
-  store_str <- store_par <- str_par_ref_list <- str_par_foc_list <-
+  store_str <- store_par <- s_p_ref_list <- s_p_foc_list <-
     vector(mode = "list", N + 1)
-  delta_h_str_vs_par_ref <- delta_h_str_vs_par_foc <- delta_h_R_vs_Ef <-
+  delta_s_p_ref <- delta_s_p_foc <- delta_h_R_Ef <-
     as.data.frame(matrix(nrow = 8, ncol = N))
-  h_R_Ef <- h_str_vs_par_ref <- h_str_vs_par_foc <-
+  h_R_Ef <- h_s_p_ref <- h_s_p_foc <- 
     as.data.frame(matrix(nrow = 8, ncol = N + 1))
-  aggregate_par <- aggregate_str <- h_aggregate_str_par <-
+  acai_p <- acai_s <- h_acai_s_p <- 
     as.data.frame(matrix(nrow = 4, ncol = N + 1))
-  h_aggregate_par <-  delta_h_str_par_aggregate <-
-    as.data.frame(matrix(nrow = 4, ncol = N))
-
+  h_acai_p <-  delta_h_s_p_acai <- as.data.frame(matrix(nrow = 4, ncol = N))
   AI_ratios <- as.data.frame(matrix(nrow = 2, ncol = N + 1))
 
   # Call PartInv with the full item set under strict invariance
-  store_str[[1]] <- PartInv(propsel = propsel,
-                            cut_z = cut_z,
-                            weights_item, weights_latent,
-                            alpha_r = alpha_r,
-                            alpha_f = alpha_f,
-                            psi_r = psi_r,
-                            psi_f = psi_f,
-                            lambda_r = lambda_f * pmix_f +
-                              lambda_r * pmix_ref,
-                            nu_r = nu_f * pmix_f + nu_r * pmix_ref,
-                            Theta_r = Theta_f * pmix_f +
-                              Theta_r * pmix_ref,
-                            pmix_ref = pmix_ref,
-                            plot_contour = plot_contour,
-                            labels = c("Reference", "Focal"),
-                            show_mi_result = show_mi_result)
-
+  store_str[[1]] <- 
+    PartInv(propsel = propsel, cut_z = cut_z, weights_item, weights_latent,
+            alpha_r = as.numeric(alpha_r), alpha_f = as.numeric(alpha_f), 
```

Comment:
Just curious: Are there situations when `alpha_r` and `alpha_f` are not numeric?